### DISCUSSION TITLE
st2packs

### DISCUSSION THREAD

**Discussion by chadpatt** [StackStorm/st2#5666] [Packs]
We are rolling out the lastest 3.7 version of Stackstorm-HA (Kubernetes) and the st2packs:builder has not been updated.  One of the key items we want is to have python3.8.  The sidecar builder isn't updated, so we are only getting python3.6.

Can the maintainer of the image please roll a current image?

**Comment by arm4b** [comment]
Thanks for raising this @chadpatt !

The root cause there looks like in this issue: https://github.com/StackStorm/st2packs-dockerfiles/issues/25
when the Docker hub removed automated builds and the repository remained without a working CI/CD pipeline.

If you could contribute the PR with the GH Actions to deploy to Docker Hub automatically on PR merge, - other maintainers can review and merge it. Help defnitely wanted there.

**Reply by chadpatt** [reply]
@armab I have absolutely no idea where to even start.  Sorry, absolutely underwater at the moment and might just have to NFS to get this back on track.

**Reply by chadpatt** [reply]
Never mind, figured it out on my end.
